### PR TITLE
remaining relevant vm fields added

### PR DESCRIPTION
**Pull request by MattBGP**

### CODE REVIEW

**Review by luthermonson — APPROVED** [review]

File: types.go [hunk 1]
```diff
@@ -290,11 +340,21 @@ const (
 )
 
 type RRDData struct {
-	MaxCPU  int
-	MaxMem  uint64
-	Disk    int
-	MaxDisk uint64
-	Time    uint64
+	Maxcpu    uint64  `json:"maxcpu"`
+	Cpu       float64 `json:"cpu"`
```
Comment (luthermonson) [inline types.go:344]: should be CPU

Comment (MattBGP) [reply types.go:344]: Alright, changing 👍

File: types.go [hunk 2]
```diff
@@ -234,6 +237,53 @@ type VirtualMachine struct {
 	HA             HA         `json:",omitempty"`
 }
 
+type Nic struct {
```
Comment (luthermonson) [inline types.go:240]: if it's an acronym try and use all caps, NICs and NIC

Comment (MattBGP) [reply types.go:240]: Alright, I was wondering while browsing through the code. Never did it like that haha..

Comment (luthermonson) [reply types.go]: just a go thing that comes in from the official style guide. https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Comment (MattBGP) [reply types.go]: Ahh I see, will look at it, Thanks!

**Comment by luthermonson** [comment]
so where did you find these other fields? I'm looking here: https://pve.proxmox.com/pve-docs/api-viewer/#/nodes/{node}/qemu/{vmid}/status/current but you seem to have found things i never noticed.

**Comment by MattBGP** [comment]
Well, the pve-docs seem to be outdated. My Node is running Proxmox 8.0.3 and has those fields.

**Comment by MattBGP** [comment]
I requested the API with Postman

**Comment by luthermonson** [comment]
could you get the data from those requests and anonymize them a bit and put it in a quick test so the unmarshalling is confirmed working? take a look at how i did the rdddata test

**Comment by luthermonson** [comment]
oh, you probably use ceph! i bet ill never see that blockstat stuff. not sure why NIC wasn't coming back. do you use SDN too?

**Comment by MattBGP** [comment]
Yes, I'll try. Codecov doesn't seem to like the Timestamp function within the struct too.

**Comment by MattBGP** [comment]
I actually never used CEPH and SDN

**Comment by luthermonson** [comment]
welp i upgraded one of my proxmox nodes to 8 and i still dont have those fields... i literally have no idea how you found them then :D whatever, extra struct fields dont hurt anyone else so they'll get merged

**Comment by luthermonson** [comment]
that one will get covered if do a test to pull a vm and assert the timestamp is equal to something. codecov is... just a suggestion ill still merge PRs with missing coverage

**Comment by luthermonson** [comment]
this should be pretty close to what you need for a test https://github.com/luthermonson/go-proxmox/blob/main/virtual_machine_test.go#L11-L24

and then just put your json in a gock call so a `vm.Ping()` populates the struct and then you can assert a couple things against it

**Comment by MattBGP** [comment]
> this should be pretty close to what you need for a test https://github.com/luthermonson/go-proxmox/blob/main/virtual_machine_test.go#L11-L24
> 
> and then just put your json in a gock call so a `vm.Ping()` populates the struct and then you can assert a couple things against it

Parsed without any errors.

**Comment by luthermonson** [comment]
Can I get the json response your getting from postman and I'll wrap a test around it. Make sure to anonymize anything that you don't want public

**Comment by MattBGP** [comment]
Below are the unmodified JSON Body from RRDDATA and Status vm.Ping()

`GET /api2/json/nodes/<node>/qemu/<vmid>/status/current`
```
{
    "data": {
        "pid": 1563102,
        "shares": 1000,
        "agent": 1,
        "diskwrite": 1515457024,
        "cpus": 8,
        "ha": {
            "managed": 0
        },
        "maxmem": 2097152000,
        "blockstat": {
            "scsi0": {
                "rd_total_time_ns": 7089432813,
                "flush_total_time_ns": 7442045713,
                "wr_total_time_ns": 65889619830,
                "failed_rd_operations": 0,
                "rd_bytes": 649448960,
                "wr_bytes": 1515457024,
                "unmap_operations": 469,
                "failed_unmap_operations": 0,
                "failed_wr_operations": 0,
                "account_failed": true,
                "invalid_unmap_operations": 0,
                "wr_operations": 157514,
                "rd_operations": 15582,
                "failed_flush_operations": 0,
                "invalid_wr_operations": 0,
                "account_invalid": true,
                "unmap_total_time_ns": 9514953,
                "unmap_merged": 0,
                "timed_stats": [],
                "unmap_bytes": 15973687808,
                "invalid_flush_operations": 0,
                "idle_time_ns": 4427685914,
                "flush_operations": 15494,
                "invalid_rd_operations": 0,
                "wr_highest_offset": 2808696832,
                "rd_merged": 0,
                "wr_merged": 0
            },
            "ide2": {
                "unmap_merged": 0,
                "timed_stats": [],
                "unmap_bytes": 0,
                "invalid_flush_operations": 0,
                "idle_time_ns": 170803536780303,
                "flush_operations": 0,
                "invalid_rd_operations": 0,
                "wr_highest_offset": 0,
                "rd_merged": 0,
                "wr_merged": 0,
                "failed_flush_operations": 0,
                "invalid_wr_operations": 0,
                "account_invalid": true,
                "unmap_total_time_ns": 0,
                "unmap_operations": 0,
                "failed_unmap_operations": 0,
                "failed_wr_operations": 0,
                "account_failed": true,
                "invalid_unmap_operations": 0,
                "wr_operations": 0,
                "rd_operations": 98,
                "rd_total_time_ns": 10689186,
                "flush_total_time_ns": 0,
                "wr_total_time_ns": 0,
                "failed_rd_operations": 0,
                "rd_bytes": 344348,
                "wr_bytes": 0
            }
        },
        "uptime": 170815,
        "cpu": 0.0112815646165076,
        "running-machine": "pc-i440fx-8.0+pve0",
        "balloon": 2097152000,
        "qmpstatus": "running",
        "status": "running",
        "maxdisk": 18467520512,
        "diskread": 649793308,
        "freemem": 887222272,
        "ballooninfo": {
            "actual": 2097152000,
            "max_mem": 2097152000,
            "free_mem": 887222272,
            "major_page_faults": 1811,
            "minor_page_faults": 3803793,
            "mem_swapped_out": 0,
            "mem_swapped_in": 0,
            "total_mem": 2015014912,
            "last_update": 1693252591
        },
        "vmid": 1001,
        "balloon_min": 2097152000,
        "mem": 1127792640,
        "proxmox-support": {
            "pbs-dirty-bitmap-savevm": true,
            "pbs-dirty-bitmap": true,
            "query-bitmap-info": true,
            "pbs-masterkey": true,
            "backup-max-workers": true,
            "pbs-dirty-bitmap-migration": true,
            "pbs-library-version": "1.4.0 (UNKNOWN)"
        },
        "running-qemu": "8.0.2",
        "name": "matt",
        "netout": 14139344,
        "netin": 547369168,
        "nics": {
            "tap1001i0": {
                "netout": 14139344,
                "netin": 547369168
            }
        },
        "disk": 0
    }
}
```
`GET /api2/json/nodes/<node>/qemu/<vmid>/rrddata?timeframe=hour&cf=AVERAGE`
```
{
    "data": [
        {
            "maxmem": 2097152000,
            "mem": 1126229879.46667,
            "time": 1693248540,
            "cpu": 0.0110221376596417,
            "maxcpu": 8,
            "netout": 79.3966666666667,
            "netin": 2820.92333333333,
            "disk": 0,
            "diskwrite": 1542.82666666667,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "maxcpu": 8,
            "netin": 2697.14,
            "netout": 64.6466666666666,
            "disk": 0,
            "maxmem": 2097152000,
            "time": 1693248600,
            "mem": 1127215718.4,
            "cpu": 0.0110793716491557,
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 9543.68
        },
        {
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 3529.38666666667,
            "maxcpu": 8,
            "disk": 0,
            "netin": 2513.07333333333,
            "netout": 65.4066666666667,
            "maxmem": 2097152000,
            "cpu": 0.0109717338452403,
            "time": 1693248660,
            "mem": 1126610261.33333
        },
        {
            "maxcpu": 8,
            "netin": 3097.47833333333,
            "netout": 210.725,
            "disk": 0,
            "maxmem": 2097152000,
            "time": 1693248720,
            "mem": 1127176669.86667,
            "cpu": 0.0113106783109343,
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 6225.92
        },
        {
            "maxmem": 2097152000,
            "cpu": 0.0114879678215619,
            "mem": 1127001292.8,
            "time": 1693248780,
            "maxcpu": 8,
            "disk": 0,
            "netout": 174.101666666667,
            "netin": 2680.885,
            "diskwrite": 13837.6533333333,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "diskwrite": 2225.49333333333,
            "maxdisk": 18467520512,
            "diskread": 0,
            "mem": 1126696345.6,
            "time": 1693248840,
            "cpu": 0.0111953822303528,
            "maxmem": 2097152000,
            "netout": 145.748333333333,
            "netin": 2783.69666666667,
            "disk": 0,
            "maxcpu": 8
        },
        {
            "diskwrite": 13537.28,
            "diskread": 0,
            "maxdisk": 18467520512,
            "maxmem": 2097152000,
            "time": 1693248900,
            "mem": 1126476936.53333,
            "cpu": 0.0114218483693642,
            "maxcpu": 8,
            "netin": 2965.43833333333,
            "netout": 184.075,
            "disk": 0
        },
        {
            "mem": 1126015453.86667,
            "time": 1693248960,
            "cpu": 0.0113001357193233,
            "maxmem": 2097152000,
            "netout": 138.846666666667,
            "netin": 3013.81833333333,
            "disk": 0,
            "maxcpu": 8,
            "diskwrite": 7249.92,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "netin": 3018.02,
            "netout": 77.6466666666667,
            "disk": 0,
            "maxcpu": 8,
            "time": 1693249020,
            "mem": 1127144721.06667,
            "cpu": 0.0110855019068012,
            "maxmem": 2097152000,
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 5092.69333333333
        },
        {
            "maxdisk": 18467520512,
            "diskread": 0,
            "diskwrite": 2177.70666666667,
            "disk": 0,
            "netout": 108.323333333333,
            "netin": 3230.395,
            "maxcpu": 8,
            "cpu": 0.0111680650240433,
            "mem": 1127015355.73333,
            "time": 1693249080,
            "maxmem": 2097152000
        },
        {
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 5331.62666666667,
            "netin": 2781.825,
            "netout": 50.75,
            "disk": 0,
            "maxcpu": 8,
            "time": 1693249140,
            "mem": 1126827963.73333,
            "cpu": 0.0108528025253175,
            "maxmem": 2097152000
        },
        {
            "maxcpu": 8,
            "netout": 85.3066666666666,
            "netin": 2967.505,
            "disk": 0,
            "maxmem": 2097152000,
            "mem": 1126920260.26667,
            "time": 1693249200,
            "cpu": 0.0110854420873042,
            "maxdisk": 18467520512,
            "diskread": 0,
            "diskwrite": 3467.94666666667
        },
        {
            "netout": 269.963333333333,
            "netin": 3352.48666666667,
            "disk": 0,
            "maxcpu": 8,
            "mem": 1127407616,
            "time": 1693249260,
            "cpu": 0.0114185655279591,
            "maxmem": 2097152000,
            "maxdisk": 18467520512,
            "diskread": 0,
            "diskwrite": 6990.50666666667
        },
        {
            "mem": 1128802781.86667,
            "time": 1693249320,
            "cpu": 0.0115239888874064,
            "maxmem": 2097152000,
            "netout": 204.173333333333,
            "netin": 3460.21666666667,
            "disk": 0,
            "maxcpu": 8,
            "diskwrite": 15837.8666666667,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "time": 1693249380,
            "mem": 1126925789.86667,
            "cpu": 0.0110203279950895,
            "maxmem": 2097152000,
            "netin": 2821.47833333333,
            "netout": 66.7866666666667,
            "disk": 0,
            "maxcpu": 8,
            "diskwrite": 1925.12,
            "diskread": 0,
            "maxdisk": 18467520512
        },
        {
            "diskwrite": 7168,
            "maxdisk": 18467520512,
            "diskread": 0,
            "mem": 1126250427.73333,
            "time": 1693249440,
            "cpu": 0.0111043463095166,
            "maxmem": 2097152000,
            "netout": 91.6416666666667,
            "netin": 2809.57166666667,
            "disk": 0,
            "maxcpu": 8
        },
        {
            "cpu": 0.0110466932054082,
            "mem": 1126370645.33333,
            "time": 1693249500,
            "maxmem": 2097152000,
            "disk": 0,
            "netout": 24.8883333333333,
            "netin": 2896.54833333333,
            "maxcpu": 8,
            "diskwrite": 8260.26666666666,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "diskwrite": 7072.42666666667,
            "diskread": 0,
            "maxdisk": 18467520512,
            "maxmem": 2097152000,
            "time": 1693249560,
            "mem": 1126547729.06667,
            "cpu": 0.0109519644342701,
            "maxcpu": 8,
            "netin": 2987.25333333333,
            "netout": 81.8733333333333,
            "disk": 0
        },
        {
            "diskwrite": 1488.21333333333,
            "maxdisk": 18467520512,
            "diskread": 0,
            "maxmem": 2097152000,
            "cpu": 0.0114180083494482,
            "mem": 1126911044.26667,
            "time": 1693249620,
            "maxcpu": 8,
            "disk": 0,
            "netout": 197.11,
            "netin": 3189.05333333333
        },
        {
            "diskwrite": 17257.8133333333,
            "maxdisk": 18467520512,
            "diskread": 0,
            "mem": 1127406318.93333,
            "time": 1693249680,
            "cpu": 0.0114163518187301,
            "maxmem": 2097152000,
            "netout": 226.116666666667,
            "netin": 2928.46333333333,
            "disk": 0,
            "maxcpu": 8
        },
        {
            "maxcpu": 8,
            "netin": 3113.81166666667,
            "netout": 95.6633333333334,
            "disk": 0,
            "maxmem": 2097152000,
            "time": 1693249740,
            "mem": 1127320780.8,
            "cpu": 0.0110430897429287,
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 5789.01333333333
        },
        {
            "cpu": 0.0109408733064343,
            "mem": 1126281352.53333,
            "time": 1693249800,
            "maxmem": 2097152000,
            "disk": 0,
            "netout": 25.0866666666667,
            "netin": 2688.07,
            "maxcpu": 8,
            "diskwrite": 10854.4,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 2000.21333333333,
            "netin": 2633.23333333333,
            "netout": 67.4733333333333,
            "disk": 0,
            "maxcpu": 8,
            "time": 1693249860,
            "mem": 1126224896,
            "cpu": 0.0109193533045582,
            "maxmem": 2097152000
        },
        {
            "maxmem": 2097152000,
            "time": 1693249920,
            "mem": 1127204864,
            "cpu": 0.0113479376051899,
            "maxcpu": 8,
            "netin": 3422.77,
            "netout": 314.088333333333,
            "disk": 0,
            "diskwrite": 9345.70666666667,
            "diskread": 0,
            "maxdisk": 18467520512
        },
        {
            "diskwrite": 12745.3866666667,
            "maxdisk": 18467520512,
            "diskread": 0,
            "maxmem": 2097152000,
            "mem": 1127593779.2,
            "time": 1693249980,
            "cpu": 0.0112358986699012,
            "maxcpu": 8,
            "netout": 134.206666666667,
            "netin": 3107.93333333333,
            "disk": 0
        },
        {
            "diskwrite": 3898.02666666667,
            "diskread": 0,
            "maxdisk": 18467520512,
            "maxmem": 2097152000,
            "cpu": 0.0109559552282692,
            "time": 1693250040,
            "mem": 1126532437.33333,
            "maxcpu": 8,
            "disk": 0,
            "netin": 3033.94833333333,
            "netout": 53.44
        },
        {
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 13994.6666666667,
            "netin": 3120.18166666667,
            "netout": 246.948333333333,
            "disk": 0,
            "maxcpu": 8,
            "time": 1693250100,
            "mem": 1127495270.4,
            "cpu": 0.0118648893240793,
            "maxmem": 2097152000
        },
        {
            "maxdisk": 18467520512,
            "diskread": 0,
            "diskwrite": 10185.3866666667,
            "maxcpu": 8,
            "netout": 78.9366666666667,
            "netin": 2788.51666666667,
            "disk": 0,
            "maxmem": 2097152000,
            "mem": 1126738397.86667,
            "time": 1693250160,
            "cpu": 0.0111245828956261
        },
        {
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 1931.94666666667,
            "maxcpu": 8,
            "netin": 2740.13833333333,
            "netout": 59.8433333333333,
            "disk": 0,
            "maxmem": 2097152000,
            "time": 1693250220,
            "mem": 1126511616,
            "cpu": 0.0113329365505118
        },
        {
            "cpu": 0.0113286971897728,
            "time": 1693250280,
            "mem": 1126446967.46667,
            "maxmem": 2097152000,
            "disk": 0,
            "netin": 2693.59166666667,
            "netout": 158.071666666667,
            "maxcpu": 8,
            "diskwrite": 8396.8,
            "diskread": 0,
            "maxdisk": 18467520512
        },
        {
            "maxcpu": 8,
            "disk": 0,
            "netin": 2752.36833333333,
            "netout": 149.161666666667,
            "maxmem": 2097152000,
            "cpu": 0.0114467719533038,
            "time": 1693250340,
            "mem": 1127275315.2,
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 16008.5333333333
        },
        {
            "disk": 0,
            "netout": 147.77,
            "netin": 2677.11666666667,
            "maxcpu": 8,
            "cpu": 0.0112991940200119,
            "mem": 1126700236.8,
            "time": 1693250400,
            "maxmem": 2097152000,
            "maxdisk": 18467520512,
            "diskread": 0,
            "diskwrite": 7311.36
        },
        {
            "maxmem": 2097152000,
            "time": 1693250460,
            "mem": 1127180492.8,
            "cpu": 0.0110320596219102,
            "maxcpu": 8,
            "netin": 2505.91666666667,
            "netout": 68.0933333333333,
            "disk": 0,
            "diskwrite": 9256.96,
            "diskread": 0,
            "maxdisk": 18467520512
        },
        {
            "maxcpu": 8,
            "disk": 0,
            "netin": 3214.18666666667,
            "netout": 65.88,
            "maxmem": 2097152000,
            "cpu": 0.0110694167178643,
            "time": 1693250520,
            "mem": 1127314158.93333,
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 1556.48
        },
        {
            "maxcpu": 8,
            "disk": 0,
            "netin": 2744.03666666667,
            "netout": 211.753333333333,
            "maxmem": 2097152000,
            "cpu": 0.0114378848988629,
            "time": 1693250580,
            "mem": 1127520119.46667,
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 18821.12
        },
        {
            "diskwrite": 10294.6133333333,
            "maxdisk": 18467520512,
            "diskread": 0,
            "maxmem": 2097152000,
            "cpu": 0.0111667492615029,
            "mem": 1127166088.53333,
            "time": 1693250640,
            "maxcpu": 8,
            "disk": 0,
            "netout": 45.2666666666667,
            "netin": 2827.42666666667
        },
        {
            "maxcpu": 8,
            "netin": 2555.84,
            "netout": 116.206666666667,
            "disk": 0,
            "maxmem": 2097152000,
            "time": 1693250700,
            "mem": 1126957192.53333,
            "cpu": 0.01116206045989,
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 13175.4666666667
        },
        {
            "mem": 1127553024,
            "time": 1693250760,
            "cpu": 0.0111572862649938,
            "maxmem": 2097152000,
            "netout": 148.183333333333,
            "netin": 2649.06333333333,
            "disk": 0,
            "maxcpu": 8,
            "diskwrite": 13994.6666666667,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 4423.68,
            "maxcpu": 8,
            "disk": 0,
            "netin": 4084.85333333333,
            "netout": 142.871666666667,
            "maxmem": 2097152000,
            "cpu": 0.011254077558411,
            "time": 1693250820,
            "mem": 1127835716.26667
        },
        {
            "maxcpu": 8,
            "netout": 157.558333333333,
            "netin": 6206.70666666667,
            "disk": 0,
            "maxmem": 2097152000,
            "mem": 1127838924.8,
            "time": 1693250880,
            "cpu": 0.0113599954187154,
            "maxdisk": 18467520512,
            "diskread": 0,
            "diskwrite": 3222.18666666667
        },
        {
            "mem": 1128348672,
            "time": 1693250940,
            "cpu": 0.0111023004240233,
            "maxmem": 2097152000,
            "netout": 41.9533333333333,
            "netin": 2491.33833333333,
            "disk": 0,
            "maxcpu": 8,
            "diskwrite": 8369.49333333333,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "mem": 1128134178.13333,
            "time": 1693251000,
            "cpu": 0.0112479040060316,
            "maxmem": 2097152000,
            "netout": 132.838333333333,
            "netin": 2657.55666666667,
            "disk": 0,
            "maxcpu": 8,
            "diskwrite": 6560.42666666667,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "diskwrite": 5550.08,
            "maxdisk": 18467520512,
            "diskread": 0,
            "cpu": 0.0111797108064618,
            "mem": 1127511040,
            "time": 1693251060,
            "maxmem": 2097152000,
            "disk": 0,
            "netout": 128.855,
            "netin": 2654.83166666667,
            "maxcpu": 8
        },
        {
            "maxdisk": 18467520512,
            "diskread": 0,
            "diskwrite": 5625.17333333333,
            "disk": 0,
            "netout": 117.1,
            "netin": 2824.75666666667,
            "maxcpu": 8,
            "cpu": 0.0112209776347412,
            "mem": 1129393356.8,
            "time": 1693251120,
            "maxmem": 2097152000
        },
        {
            "diskwrite": 16915.2387878788,
            "maxdisk": 18467520512,
            "diskread": 0,
            "maxmem": 2097152000,
            "cpu": 0.0113053957734911,
            "mem": 1128874598.4,
            "time": 1693251180,
            "maxcpu": 8,
            "disk": 0,
            "netout": 132.723333333333,
            "netin": 2795.87378787879
        },
        {
            "cpu": 0.0111553588290355,
            "time": 1693251240,
            "mem": 1127833668.26667,
            "maxmem": 2097152000,
            "disk": 0,
            "netin": 3038.14621212121,
            "netout": 87.5333333333333,
            "maxcpu": 8,
            "diskwrite": 6261.29454545454,
            "diskread": 0,
            "maxdisk": 18467520512
        },
        {
            "diskwrite": 4990.29333333333,
            "maxdisk": 18467520512,
            "diskread": 0,
            "maxmem": 2097152000,
            "cpu": 0.0110010482445049,
            "mem": 1127526400,
            "time": 1693251300,
            "maxcpu": 8,
            "disk": 0,
            "netout": 118.275,
            "netin": 2714.615
        },
        {
            "netin": 3427.085,
            "netout": 71.595,
            "disk": 0,
            "maxcpu": 8,
            "time": 1693251360,
            "mem": 1128350924.8,
            "cpu": 0.0111266310924798,
            "maxmem": 2097152000,
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 7543.46666666667
        },
        {
            "maxmem": 2097152000,
            "cpu": 0.0111296708938305,
            "time": 1693251420,
            "mem": 1130184089.6,
            "maxcpu": 8,
            "disk": 0,
            "netin": 2656.88,
            "netout": 136.273333333333,
            "diskwrite": 5208.74666666667,
            "diskread": 0,
            "maxdisk": 18467520512
        },
        {
            "diskwrite": 5645.65333333333,
            "maxdisk": 18467520512,
            "diskread": 0,
            "maxmem": 2097152000,
            "mem": 1128423765.33333,
            "time": 1693251480,
            "cpu": 0.0111255240030073,
            "maxcpu": 8,
            "netout": 29.4933333333333,
            "netin": 3085.67333333333,
            "disk": 0
        },
        {
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 5495.46666666667,
            "disk": 0,
            "netin": 3518.03666666667,
            "netout": 66.54,
            "maxcpu": 8,
            "cpu": 0.0113824267643594,
            "time": 1693251540,
            "mem": 1127842816,
            "maxmem": 2097152000
        },
        {
            "diskwrite": 7714.13333333333,
            "diskread": 0,
            "maxdisk": 18467520512,
            "time": 1693251600,
            "mem": 1128894532.26667,
            "cpu": 0.0112408034403697,
            "maxmem": 2097152000,
            "netin": 3023.24333333333,
            "netout": 126.3,
            "disk": 0,
            "maxcpu": 8
        },
        {
            "diskwrite": 8260.26666666666,
            "diskread": 0,
            "maxdisk": 18467520512,
            "maxmem": 2097152000,
            "cpu": 0.0113451495083074,
            "time": 1693251660,
            "mem": 1129145139.2,
            "maxcpu": 8,
            "disk": 0,
            "netin": 3200.65166666667,
            "netout": 93.3266666666666
        },
        {
            "cpu": 0.0113453917711096,
            "mem": 1128183603.2,
            "time": 1693251720,
            "maxmem": 2097152000,
            "disk": 0,
            "netout": 156.328333333333,
            "netin": 3104.86166666667,
            "maxcpu": 8,
            "diskwrite": 11919.36,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "maxmem": 2097152000,
            "cpu": 0.0132748487401862,
            "mem": 1128778820.26667,
            "time": 1693251780,
            "maxcpu": 8,
            "disk": 0,
            "netout": 60.0483333333333,
            "netin": 3049.43333333333,
            "diskwrite": 10608.64,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "maxdisk": 18467520512,
            "diskread": 0,
            "diskwrite": 8669.86666666666,
            "maxcpu": 8,
            "netout": 80.66,
            "netin": 3509.53666666667,
            "disk": 0,
            "maxmem": 2097152000,
            "mem": 1129246856.53333,
            "time": 1693251840,
            "cpu": 0.0112137320049851
        },
        {
            "netout": 74.8033333333334,
            "netin": 3144.20166666667,
            "disk": 0,
            "maxcpu": 8,
            "mem": 1128952285.86667,
            "time": 1693251900,
            "cpu": 0.0111774124687674,
            "maxmem": 2097152000,
            "maxdisk": 18467520512,
            "diskread": 0,
            "diskwrite": 4642.13333333333
        },
        {
            "diskwrite": 19756.3733333333,
            "maxdisk": 18467520512,
            "diskread": 0,
            "maxmem": 2097152000,
            "mem": 1129150737.06667,
            "time": 1693251960,
            "cpu": 0.0117926478527666,
            "maxcpu": 8,
            "netout": 241.37,
            "netin": 3481.255,
            "disk": 0
        },
        {
            "disk": 0,
            "netin": 3766.27666666667,
            "netout": 69.7016666666667,
            "maxcpu": 8,
            "cpu": 0.0111585671952275,
            "time": 1693252020,
            "mem": 1128763050.66667,
            "maxmem": 2097152000,
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 3932.16
        },
        {
            "diskwrite": 17476.2666666667,
            "diskread": 0,
            "maxdisk": 18467520512,
            "maxmem": 2097152000,
            "cpu": 0.0112721935194054,
            "time": 1693252080,
            "mem": 1128643652.26667,
            "maxcpu": 8,
            "disk": 0,
            "netin": 3108.53333333333,
            "netout": 123.841666666667
        },
        {
            "diskwrite": 2778.45333333333,
            "maxdisk": 18467520512,
            "diskread": 0,
            "maxmem": 2097152000,
            "cpu": 0.0111754933722218,
            "mem": 1128513877.33333,
            "time": 1693252140,
            "maxcpu": 8,
            "disk": 0,
            "netout": 67.3433333333333,
            "netin": 2954.41166666667
        },
        {
            "netin": 2983.06833333333,
            "netout": 82.3166666666666,
            "disk": 0,
            "maxcpu": 8,
            "time": 1693252200,
            "mem": 1128380142.93333,
            "cpu": 0.0113860832760714,
            "maxmem": 2097152000,
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 5563.73333333333
        },
        {
            "maxcpu": 8,
            "disk": 0,
            "netout": 73.36,
            "netin": 4095.63666666667,
            "maxmem": 2097152000,
            "cpu": 0.0113119362659409,
            "mem": 1128338432,
            "time": 1693252260,
            "maxdisk": 18467520512,
            "diskread": 0,
            "diskwrite": 1679.36
        },
        {
            "maxmem": 2097152000,
            "cpu": 0.0113734092652031,
            "time": 1693252320,
            "mem": 1128399052.8,
            "maxcpu": 8,
            "disk": 0,
            "netin": 3108.24,
            "netout": 212.55,
            "diskwrite": 5857.28,
            "diskread": 0,
            "maxdisk": 18467520512
        },
        {
            "cpu": 0.0120984402163308,
            "mem": 1129448584.53333,
            "time": 1693252380,
            "maxmem": 2097152000,
            "disk": 0,
            "netout": 475.106666666667,
            "netin": 3052.17,
            "maxcpu": 8,
            "diskwrite": 23947.9466666667,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "maxmem": 2097152000,
            "time": 1693252440,
            "mem": 1128350173.86667,
            "cpu": 0.0114016215025843,
            "maxcpu": 8,
            "netin": 3214.62333333333,
            "netout": 148.906666666667,
            "disk": 0,
            "diskwrite": 11864.7466666667,
            "diskread": 0,
            "maxdisk": 18467520512
        },
        {
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 2607.78666666667,
            "disk": 0,
            "netin": 3346.48,
            "netout": 80.0466666666667,
            "maxcpu": 8,
            "cpu": 0.011339877668421,
            "time": 1693252500,
            "mem": 1128205038.93333,
            "maxmem": 2097152000
        },
        {
            "mem": 1127855240.53333,
            "time": 1693252560,
            "cpu": 0.0112381427344003,
            "maxmem": 2097152000,
            "netout": 70.5866666666666,
            "netin": 2925.52333333333,
            "disk": 0,
            "maxcpu": 8,
            "diskwrite": 6075.73333333333,
            "maxdisk": 18467520512,
            "diskread": 0
        },
        {
            "diskread": 0,
            "maxdisk": 18467520512,
            "diskwrite": 6048.42666666667,
            "disk": 0,
            "netin": 2616.26833333333,
            "netout": 45.635,
            "maxcpu": 8,
            "cpu": 0.0110708128990444,
            "time": 1693252620,
            "mem": 1127807863.46667,
            "maxmem": 2097152000
        },
        {
            "diskwrite": 7536.64,
            "diskread": 0,
            "maxdisk": 18467520512,
            "time": 1693252680,
            "mem": 1128515242.66667,
            "cpu": 0.0114237931879527,
            "maxmem": 2097152000,
            "netin": 3528.83666666667,
            "netout": 191.973333333333,
            "disk": 0,
            "maxcpu": 8
        }
    ]
}
```

**Comment by luthermonson** [comment]
great! thanks for this

**Pull request merged**